helpers/constants.js

```diff
@@ -28,10 +28,10 @@ const RATING_1_SECRET = web3Utils.soliditySha3(RATING_1_SALT, MANAGER_RATING);
 const RATING_2_SECRET = web3Utils.soliditySha3(RATING_2_SALT, WORKER_RATING);
 
 module.exports = {
-  MANAGER: MANAGER || web3.eth.accounts[0],
-  EVALUATOR: EVALUATOR || web3.eth.accounts[1],
-  WORKER: WORKER || web3.eth.accounts[2],
-  OTHER: OTHER || web3.eth.accounts[3],
+  MANAGER: MANAGER || "0xb77D57F4959eAfA0339424b83FcFaf9c15407461",
+  EVALUATOR: EVALUATOR || "0x9dF24e73f40b2a911Eb254A8825103723E13209C",
+  WORKER: WORKER || "0x27fF0C145E191C22C75cD123C679C3e1F58a4469",
+  OTHER: OTHER || "0x0021Cb24d7D4e669120b139030095315DFa6699a",
```

Comment:
Web3 got rid of sync methods. I don't know if this actually works for the Parity setup. Might require @area's `requestSync` wizardry.